docs/link_to_download_example.mdx

```diff
@@ -0,0 +1,7 @@
+---   
+title: Example with downloadable spreadsheet
+---   
+      
+import useBaseUrl from '@docusaurus/useBaseUrl';
+    
+<a target="_blank" rel="noopener noreferrer" href={useBaseUrl('/associations.csv')}>Download Spreadsheet</a>
```

Comment:
rel="noopener noreferrer" to address a "reverse tabnabbing attack" (great phrase).